matrix/count_islands_in_matrix.py

```diff
@@ -4,20 +4,21 @@
 
 
 class matrix:  # Public class to implement a graph
-    def __init__(self, row: int, col: int, graph: list):
+    def __init__(self, row: int, col: int, graph: list[list[bool]]) -> None:
```

Comment:
Does this need to indicate that it returns `None`? That's pretty standard

Reply:
Yes, it does need to indicate.